### DISCUSSION TITLE
All the ServerName functions are undefined in latest 18.4.0 release.

### DISCUSSION THREAD

**Discussion by Napzu** [uNetworking/uWebSockets.js#359] [Q&A] [answered]
missingServerName, addServerName and removeServerName functions are undefined in latest 18.4.0 release.

**Comment by ghost** [comment]
How do you mean "undefined"? I know people who use them.

**Comment by Napzu** [comment]
Well when I try to test SNI funtions, node crashes and tells me me reason, which is that the called function is undefined.

**Comment by ghost** [comment]
You need to specify more info. I cannot infer anything from what you are telling me. Is the function "undefined" as in JavaScript, or is the function an undefined reference as in C++ linking issues? What platform? What reproduces it? Etc, etc, etc.

**Comment by Napzu** [comment]
My server: CentOS 8, node v12.11.1

Testing with [ServerName.js](https://github.com/uNetworking/uWebSockets.js/blob/master/examples/ServerName.js) example.

After runninng "node index.js" gives this:
![image](https://user-images.githubusercontent.com/23162922/92013102-fe240d00-ed55-11ea-8c1c-6face01b148f.png)

**Comment by wight-airmash** [comment]
The thing is probably that the 18.4.0 release doesn't include these features, but if you take the binaries branch, the example will work.

**Comment by ghost** [comment]
SNI is not in any release yet it is still under development

**Comment by ghost** [comment, marked as answer]
18.5 is released with this
